apps/native-component-list/screens/DocumentPickerScreen.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Alert, Image, Text, View } from 'react-native';
+import { Alert, Image, Text, View, Switch } from 'react-native';
```

Comment:
revert changes to this file?

Reply:
Why? It was intended to be part of this PR. It let me test an issue that was present in previous versions of SDK, so I thought that adding it might help with preventing regression :)

Reply:
Actually I think this file was part of a different issue, but let's leave it 🙂